### PR TITLE
[TravisCI] Add ILP64 build with cmake.

### PR DESCRIPTION
**Pull request by fghoussen**
## Pull request purpose

Add cmake ILP64 build. "Should" fail because of ICB : this is #230... If KO, I'll suppress ICB temporarily to keep testing the most possible things step-by-step...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -225,6 +225,46 @@ jobs:
       sudo docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp                                                               \
       &&                                                                                                                \
       sudo docker start -a mobydick
+  - stage: interface64
```
Comment (sylvestre) [inline .travis.yml]: same as the other PR, lot of code duplication here :/

Comment (fghoussen) [reply .travis.yml]: No sure to get what you mean: AFAIK, in yml, no way to "factorize" code or "make functions", or, I just don't know how to do that. You need to add job stage by stage as yml grammar impose it, or, you could put all jobs in one stage but then all config are mixed (difficult to debug when pb shows up) and huge-logs blow up (can't access traces when pb show up)

**Comment by fghoussen** [comment]
Yet another problem ?!... CI freezes for ilp64 cmake : no idea why ? Trying static build

**Comment by fghoussen** [comment]
Great : BLA_STATIC finds .so ?!... Build OK but run freezes ?!...

**Comment by fghoussen** [comment]
Will try a few tricks and give up if this is too tricky to get done : could be endless...

**Comment by fghoussen** [comment]
WTF ? 3dd5ac6 cancels out 72cc5e3 CI ?!... can restart 72cc5e3 jobs. will test one after the other

File: .travis.yml (hunk 1 above)
Comment (sylvestre) [reply .travis.yml]: most of the operations should be moved into a script
link "setup-env.sh" and called with a bash -v

**Comment by fghoussen** [comment]
Cmake still "decide" to link with ` *lp64.so ` not **i**lp64 ?!... Try 2 more tricks and give up....

**Comment by fghoussen** [comment]
` BLA_VENDOR='Intel10_64ilp' cmake -D INTERFACE64=ON -D EXAMPLES=ON -D ICB=ON -D MPI=ON .. ` just  work fine on my debian box but fails on CI (build OK, running tests freeze ) ?! Will try a few more trick and give up...

**Comment by fghoussen** [comment]
WTF ?! Endless burden : giving up.

**Pull request closed without merging**